src/components/form/range/dual_range.tsx

```diff
@@ -78,10 +78,18 @@ export class EuiDualRangeClass extends Component<
   private dragAcc = 0;
 
   get lowerValue() {
-    return this.props.value ? this.props.value[0] : this.props.min;
+    if (this.props.value && this.props.value[0] !== '') {
+      return this.props.value[0];
+    }
+
+    return this.props.min;
```

Comment:
Can we write this a bit more succinctly?

```suggestion
    return this.props.value?.[0] || this.props.min;
```

Reply:
oh wait, i see, this will fall through incorrectly for values of 0. thoughts on this syntax instead?

```suggestion
    const lowerValue = this.props.value?.[0];
    return lowerValue && isNaN(lowerValue) ? lowerValue : this.props.min;
```

Reply:
Yeah I thought about it when I wrote this piece and but chose my version for easier readability. I like that `isNaN` handles value coercing and validation for us here though. I'll update my code!

Reply:
I ultimately had to replace it with `return (lowerValue || lowerValue === 0 || lowerValue === '0') && !isNaN(Number(lowerValue)) ? lowerValue : this.props.min;` to handle all the cases and not have to disable eqeqeq eslint rule if I used coercion.

Reply:
I'm testing this a bit more and unfortunately this still just doesn't work like normal inputs should if the user wants to clear both inputs. Attempting to clear one autofills the other:

![screencap](https://github.com/elastic/eui/assets/549407/02c9ff78-a582-493a-8d37-549f12f0f917)

I also hadn't realized values could be strings as well as numbers, apologies for that mistaken impression on my end. I took a step back to look at the original report/request and I have some concerns that they're attempting to do is simply not something EuiDualRange should do or support. Let's hold on a tick to hear back from Hannah.